datafusion/sql/src/unparser/expr.rs

```diff
@@ -391,7 +391,9 @@ impl Unparser<'_> {
             Expr::ScalarVariable(_, _) => {
                 not_impl_err!("Unsupported Expr conversion: {expr:?}")
             }
-            Expr::IsNull(_) => not_impl_err!("Unsupported Expr conversion: {expr:?}"),
+            Expr::IsNull(expr) => {
+                Ok(ast::Expr::IsNull(Box::new(self.expr_to_sql(expr)?)))
+            }
             Expr::IsNotFalse(_) => not_impl_err!("Unsupported Expr conversion: {expr:?}"),
```

Comment:
Ah it looks like I maybe mis read the code originally and the other missing expression is `Expr::IsNotFalse` (rather than `Expr::IsNotNull`)

Reply:
Thanks for reviewing! I think I can fix `IsNotFalse` in another PR. What do you think?

Reply:
I think a second PR will be great